k2/util/Logging.h

```diff
@@ -0,0 +1,32 @@
+//
```

Comment:
the first line of every file should be the path to the file.

Reply:
Done